src/components/Success/Success.js

```diff
@@ -0,0 +1,18 @@
+import { usePopups } from '../../contexts/PopupContext';
+import PopupWithForm from '../PopupWithForm/PopupWithForm';
+
+const Success = ({ showSignIn }) => {
+  const [popupState] = usePopups();
+  return (
+    <PopupWithForm
+      hideForm={true}
+      formName='success'
+      isOpen={popupState.isSuccessPopupOpen}
+      title='Registration successfully completed!'
+      redirectText='Sign in'
+      handleRedirect={showSignIn}
+    ></PopupWithForm>
```

Comment:
**Might be better**


you can use `self-closing` tags if the component doesn't have `children`